core/file_extension/file_extension.py

```diff
@@ -32,6 +33,22 @@
     "tot csv": ".csv",
     "dot cassie": ".csv",
     "dot text": ".txt",
+    "dot julia": ".jl",
+    "dot J L": ".jl",
+    "dot html": ".html",
+    "dot css": ".css",
+    "dot sass": ".sass",
+    "dot svg": ".svg",
+    "dot png": ".png",
+    "dot wave": ".wav",
+    "dot flack": ".flack",
```

Comment:
Was this mean to be `.flac`?

Reply:
Whoops, missed that too :-(

Reply:
https://github.com/knausj85/knausj_talon/pull/1076